### DISCUSSION TITLE
Dev-new-issue

### DISCUSSION THREAD

**Discussion by ami-spark** [microsoft/vscode-discussions#697] [Extension Development QnA]
{
	"name": "Code - OSS with VNC",

	// Image contents: https://github.com/microsoft/vscode-dev-containers/blob/master/repository-containers/images/github.com/microsoft/vscode/.devcontainer/base.Dockerfile
	"image": "mcr.microsoft.com/vscode/devcontainers/repos/microsoft/vscode:branch-main",
	"overrideCommand": false,
	"runArgs": [ "--init", "--security-opt", "seccomp=unconfined", "--shm-size=1g"],

	// VS Code extensions and settings
	"customizations": {
		"vscode": {
			"settings": {
				"resmon.show.battery": false,
				"resmon.show.cpufreq": false
			},
			"extensions": [
				"dbaeumer.vscode-eslint",
				"mutantdino.resourcemonitor"
			]
		}
	},

	// noVNC, VNC
	"forwardPorts": [6080, 5901],
	"portsAttributes": {
		"6080": {
			"label": "VNC web client (noVNC)",
			"onAutoForward": "silent"
		},
		"5901": {
			"label": "VNC TCP port",
			"onAutoForward": "silent"
		}
	},

	// Optionally loads a cached yarn install for the repo
	"postCreateCommand": ".devcontainer/prebuilt/cache/restore-diff.sh",

	"remoteUser": "node",

	"hostRequirements": {
		"memory": "9gb"
	}
}

**Comment by burkeholland** [comment]
Can you please add your question? This is just a code block.

**Reply by ami-spark** [reply]
emoji template has default old extensions; won't be transformed in the CI Azure pipeline as tested in docker after script manipulation.
Happy coding!
amitpaulom@outlook.com

**Comment by isidorn** [comment]
Closing for lack of clarity.

**Comment by ami-spark** [comment]
@Isidoor
Please add me in looping for further ticket resolve in CI

Thanks
Amit
***@***.***
